descriptors/ModuleDescriptor-template.json

```diff
@@ -459,8 +506,7 @@
         "invoice-storage.invoice-lines.item.delete",
         "invoice-storage.invoice-number.get",
         "invoice-storage.invoice-line-number.get",
-        "voucher-storage.module.all",
-        "batch-group-storage.module.all"
```

Comment:
Please, remove this back. Because it is already implemented interface - batch-group-storage and add here entire set `batch-voucher-storage.module.all`.